components/Calender.tsx

```diff
@@ -25,23 +25,21 @@ export const Calender = ({ pathId, streak, onStreakUpdate }: Props) => {
 
     const sortedDates = dates
       .map((dateStr) => dayjs(dateStr, 'D-MMMM-YYYY'))
-      .sort((a, b) => a.diff(b, 'day'));
+      .sort((a, b) => b.diff(a, 'day'));
```

Comment:
Can we use better variable names, something that describe what a and b are here.